### PR TITLE
New Houses

### PR DESCRIPTION
**Pull request by TinyWolfGirl**
While putting my [map editor](https://github.com/KrazyTheFox/Cataclysm-DDA-Map-Editor) through its paces and ironing out some bugs I've created 6 houses that are game-ready. Thought I'd toss them into a PR before the next release.

You can see the renders of each house here: http://imgur.com/a/MvEyC

### CODE REVIEW

**Comment by Dozed12** [comment]
I'm really excited to see your editor, I wish we had a bounty for creating such tool because that looks perfect.

**Comment by KrazyTheFox** [comment]
There's nothing stopping you from making one. ;)

**Comment by KA101** [comment]
I like what I see here.  Not sure where the freeze is just yet, though.

**Comment by KrazyTheFox** [comment]
That's fine; I'm in no hurry. This release or the next one—it doesn't matter to me.

**Comment by Lain-** [comment]
Though it would be nice if we can have this in before the stable though! *wink wink*

**Comment by kevingranade** [comment]
If it's strictly new building layouts it should be safe.

**Comment by KA101** [comment]
With that in mind, queued.

File: data/json/mapgen/house12.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                { "item": "magazines", "chance": 25, "x": [ 3, 4 ],  "y": 17 },
+                { "item": "homebooks", "chance": 25, "x": [ 3, 4 ],  "y": 13 },
+                { "item": "bed", "chance": 80, "x": [ 18, 19 ],  "y": [ 7, 8 ] },
+                { "item": "kitchen", "chance": 75, "x": 11,  "y": [ 15, 18 ] },
+                { "item": "bedroom", "chance": 80, "x": [ 15, 19 ],  "y": [ 6, 12 ] },
+                { "item": "livingroom", "chance": 80, "x": [ 3, 7 ],  "y": [ 13, 19 ] },
+                { "item": "cleaning", "chance": 75, "x": [ 15, 16 ],  "y": [ 17, 18 ] },
+                { "item": "dresser", "chance": 70, "x": 14,  "y": [ 14, 15 ] },
+                { "item": "kitchen", "chance": 75, "x": [ 8, 10 ],  "y": [ 13, 19 ] },
+                { "monster": "GROUP_ZOMBIE", "chance": 2, "x": [ 2, 21 ],  "y": [ 2, 21 ] }
```
Comment (KA101) [inline data/json/mapgen/house12.json]: This house is kinda profitable, I think: piles of stuff and low chance for zeds.  Gonna tweak this in the merge.

**Comment by KA101** [comment]
Yeah, on further review, these are pretty profitable houses.  They'll make it in but with a downtune on the loot spawn.

**Comment by KrazyTheFox** [comment]
Fair enough. It's not exactly clear how the item groups work, even with the documentation (that I've found, that is) and it's certainly not my intention to create profitable houses. I'll probably dig into the code sometime before I PR another set of buildings to figure all that out.

**Comment by KA101** [comment]
Perhaps I lucked into several well-stocked houses in testing, too.  I think having a lot of dressers, plus the washer/dryers with clothing spawns, made the places seem packed--same with a few bathrooms that had several soft-drug spawns.

I added more zeds because I'd not encountered any, and IMO there's always room for a few more hostiles.

**Comment by KrazyTheFox** [comment]
Good points. I'll keep those in mind for next time, thanks!

**Pull request merged**